firefox-code-search/README.md

```diff
@@ -0,0 +1,13 @@
+# firefox-code-search
+
+## What it does
+
+This extension allows you to search the firefox codebase using the awesome bar.
+
+To test it out, type 'cs' into the awesome bar followed by a search string (e.g. `cs hello world`). The results will be shown as suggestions, and clicking on a suggestion will navigate to the file where the result was found.
+
+To search a specific file, use the "path:" prefix (e.g. `cs hello path:omnibox.js`)
+
+## What it shows
+
+How to use the omnibox API to create an extension which provides suggestions asynchronously.
```

Comment:
It might be better to say something like "How to use the omnibox API to add custom suggestions to the awesome bar".